joeynmt/prediction.py

```diff
@@ -382,76 +388,39 @@ def test(cfg_file,
             logger.info("Translations saved to: %s", output_path_set)
 
 
-def translate(cfg_file: str,
-              ckpt: str,
-              output_path: str = None,
-              batch_class: Batch = Batch,
-              n_best: int = 1) -> None:
+def load_params_for_prediction(cfg_file,ckpt: str):
```

Comment:
Please add documentation + typing for what this function returns. Thanks!

Reply:
It's important to clarify that the function loads the hyperparameters from the config AND loads the model (parameters).